src/background/services/paymentManager.ts

```diff
@@ -150,6 +150,18 @@ export class PaymentManager {
     return this.sessions.filter((s) => s.isUsable);
   }
 
+  // bugfix #1071
+  get minSendAmount(): bigint {
+    // Check to see if there are any payable sessions
+    if (!this.payableSessions.length) return 0n;
+
+    // NOTE: How is a payable session defined, i.e. are could the minSendAmount be different currencies?
```

Comment:
payable session is defined here:

https://github.com/interledger/web-monetization-extension/blob/893bf23e5b777d0c8cee01b31d91eef4429d3aba/src/background/services/paymentManager.ts#L149-L151